### PR TITLE
 Identical Words

### PR DESCRIPTION
**Pull request by Amanduttbhagat**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

- Identical_Words : 
    We will give 3 Strings as input and we will check if it is possible to make 
    second_string and third_string equivalent to first_string by re-arranging characters of
    second_string and third_string any number of times

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#4480

### CODE REVIEW

**Comment by Amanduttbhagat** [comment]
@ankitaggarwal23 @HarshCasper @siddharth25pandey 
Can you review this please?

Thank you

File: Java/README.md [hunk 1]
```diff
@@ -130,7 +131,7 @@ _add list here_
 - [Linked List ~ Reverse](ds/linkedListReverse.java)
 - [Matrix Sorting](ds/MatrixSorting.java)
 - [Merge Two Sorted Arrays](ds/MergeSortedArrays.java)
-- [Middle Of LinkedList](ds/MiddleOfLinkedList.java) 
+- [Middle Of LinkedList](ds/MiddleOfLinkedList.java)
```
Comment (siddharth25pandey) [inline Java/README.md:134]: Remove the Redundant line

File: Java/cp/Identical_Words.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+			String s2 = Identical_Words.string_sort(second_string);
+			String s3 = Identical_Words.string_sort(third_string);
+			//Checking if Strings are equal or not
+			if(s1.equals(s2) && s1.equals(s3)) {
+				System.out.println("Possible");
+			}else {
+				System.out.println("Not Possible");
+			}
+		}else {
+			System.out.println("Not Possible");
```
Comment (siddharth25pandey) [inline Java/cp/Identical_Words.java]: Please write complete print message details, What is possible or not possible .

**Review by siddharth25pandey — CHANGES_REQUESTED** [review]
@Amanduttbhagat Do the suggested changes

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by HarshCasper** [comment]
Please make an issue here and raise a PR, for an article as part of our Docs repository: https://github.com/TesseractCoding/NeoAlgo-Docs

It would be much appreciated. Thanks!

**Pull request merged**